combat-trainer.lic

```diff
@@ -774,6 +773,8 @@ class SpellProcess
     @equipment_manager = equipment_manager
     echo('New SpellProcess') if $debug_mode_ct
 
+    @settings = get_settings
```

Comment:
We don't want to call `get_settings` here as it will be missing any args that were passed to `combat-trainer`. SpellProcess should be passed `settings` as an argument, which would be what we want to store.